core/src/main/scala/cats/syntax/all.scala

```diff
@@ -94,3 +95,5 @@ trait AllSyntaxBinCompat4
 trait AllSyntaxBinCompat5 extends ParallelBitraverseSyntax
 
 trait AllSyntaxBinCompat6 extends ParallelUnorderedTraverseSyntax
+
+trait AllSyntaxBinCompat7 extends ParallelFoldMapASyntax
```

Comment:
We shouldn't need new `BinCompat` traits now that we've dropped 2.11 on master. You should be able to add `extends ParallelFoldMapASyntax` directly to the end of the `trait AllSyntax` stack.

Reply:
Done.